salesagility/SuiteCRM
# Fixed #7759 and #8273 double Compose button in subpanels

## Description
This re-applies a fix Dillon had done years ago: https://github.com/salesagility/SuiteCRM/pull/6160/files

This somehow got partially reverted in this big commit here https://github.com/salesagility/SuiteCRM/commit/acf296012d9c147f526144ee11ba2188627e44b8

I also took the opportunity to add an assignment that prevents a JS warning later on, inside the Compose window, in some cases. 

## How To Test This
1. In **Admin / System settings**, select **`Display actions within menus`**
2. then go look at any **Activities subpanel** (Leads, Accounts, Contacts, ...)
3. Confirm there is only one, correctly-styled, **Compose email** button
4. Click the Compose button to see if it works

## Types of changes

### Final checklist
